### DISCUSSION TITLE
Indicate the location of (not visible) favorites

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#113] [Q&A] [answered]
When selecting a favorite, the map is shown with its position in the center of the display, but without any indication: 
<img width="400" src="https://user-images.githubusercontent.com/121806238/229606690-c2e26c96-c4d8-45c3-95da-7f119f49a398.png"/>
Whereby with a default zoom of 14 it is not exactly recognizable where the position is located - by zooming it can of course be narrowed down.

Wouldn't it make sense to always display the address of the position and thus simplify further processing (set as destination, for example)? Like this:
<img width="400" src="https://user-images.githubusercontent.com/121806238/229606735-42cf15c0-08f6-4b69-a414-8b9fe1117ce6.png"/>

Thank you for a kind review ...

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the suggestion!

> When selecting a favorite

How? With the search favorites or the favorites list?

> the map is shown with its position in the center of the display, but without any indication

Are the favorites been set as visible in the map settings?

> Wouldn't it make sense to always display the address of the position and thus simplify further processing (set as destination, for example)?

Wouldn't this be duplicate layers?

If the favorites are set to be visible,
then the map zooms on the selected favorite and with long press, can use it in routing?

**Reply by WolfgangZ42** [reply]
Ooops - my mistake! I forgot the settings/map/layers/favorites switch - I've never used it to avoid this:
<img width="400" src="https://user-images.githubusercontent.com/121806238/229702793-efd38f15-bdb0-4883-8f3f-a503deb51acd.png"/>
(Yes - call me a purist ... 😉)

Couldn't be that easy, but is it possible to show in this case the favorite sign regardless the settings? If not, I'll rethink my behaviour ...

**Reply by devemux86** [reply]
> Couldn't be that easy, but is it possible to show in this case the favorite sign regardless the settings?

Indeed, we can improve the workflow here, in case favorites are set to be hidden.

Favorites will change a lot in the near future as I am working on favorite categories:
- https://github.com/devemux86/cruiser/discussions/45

One of the upcoming features could be changing the visibility of favorites by category. :slightly_smiling_face:

**Comment by devemux86** [comment]
The blue address marker appears when we search and want to mark a location on the map.
For example, search for an address or a point of interest.

Favorites already have markers, so the map just needs to move / zoom on the existing marker.

**Comment by WolfgangZ42** [comment]
I'm not using favorites very often, so I discovered just today, that the map doesn't change anymore when selecting a favorite. Map was centered two weeks ago - see first post of this thread. 
By playing around I figured out, that it does now centering only if "show favorites on map" is on. 
I can (or must) live with it, no question.

**Reply by devemux86** [reply]
@WolfgangZ42 currently the favorites must be visible to search / focus on them,
until we have a different implementation for their indication when they are hidden.

Previously when favorites were hidden the map would focus on the empty space,
not very useful and could be confusing to users?

**Comment by devemux86** [comment, marked as answer]
With new favorites coming in next Cruiser, let's discuss their search in that thread:
- https://github.com/devemux86/cruiser/discussions/45#discussioncomment-5792896
